app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -75,17 +96,19 @@ export default function DifferentialExpressionPanel({
             data-original-title="Download all DE genes data for this group"
           >
             <FontAwesomeIcon icon={faDownload}/></a> */}
-          <a href="https://singlecell.zendesk.com/hc/en-us/articles/6059411840027"
-            target="_blank"
-            data-analytics-name="differential-expression-docs"
-            // style={{ 'marginLeft': '10px' }} // Dev placeholder for when download is enable
-            data-toggle="tooltip"
-            data-original-title="Learn about SCP DE genes analysis"
-          >
-            <FontAwesomeIcon
-              className="action help-icon" icon={faInfoCircle}
-            />
-          </a>
+          { deModalHelpLink }
+          <Modal
+            id="de-info-modal"
+            show={showDeModal}
+            onHide={() => closeModal(setShowDeModal)}
+            animation={false}>
+            <Modal.Header>
+              <h4 className="text-center">Learn about SCP exploratory differential gene expression analysis</h4>
```

Comment:
With the popover covering a lot more content now, less prelude suffices:

```suggestion
              <h4 className="text-center">Exploratory differential expression</h4>
```